proto/elys/amm/pool.proto

```diff
@@ -5,15 +5,20 @@ option go_package = "github.com/elys-network/elys/x/amm/types";
 import "elys/amm/pool_params.proto"; 
 import "elys/amm/pool_asset.proto"; 
 import "gogoproto/gogo.proto"; 
-import "cosmos/base/v1beta1/coin.proto"; 
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
 
 message Pool {
```

Comment:
I thought that we would have multiplier per pool. Could you clarify with Dream about this?